.github/workflows/test-coverage.yaml

```diff
@@ -31,23 +31,7 @@ jobs:
         run: |
           go test ./... -coverprofile coverage.out -covermode count
           go tool cover -func coverage.out
-
-      - name: Quality Gate - Test coverage should be above threshold
-        env:
-          TESTCOVERAGE_THRESHOLD: 50
-        run: |
-          echo "Quality Gate: checking if test coverage is above threshold ..."
```

Comment:
Will these be enabled later? Or do we want to enable this coverage threshold with a low value - say 30%?

Reply:
This has now moved to `CodeCov`. It is doing to use the base commit coverage as the base and use the specified threshold for leniency. In our current config, the PR should not reduce the coverage percentage, however, the actual enforcement will not be done since everything is `informational`. 

We will enable informational with time as the team gets comfortable with `CodeCov`.

More details here - https://docs.codecov.com/docs/commit-status#target

Reply:
coo, thanks